Changelog.md

```diff
@@ -228,25 +228,25 @@
     * [Pull request 661](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-requests/661)
     * [Pull request 663](https://osrf-migration.github.io/sdformat-gh-pages/#!/osrf/sdformat/pull-requests/663)
 
-1. Add functionality to generate aggregated SDFormat descriptions via CMake.
```

Comment:
This is referring to the spec, so it should be SDFormat

Reply:
Done.